scene/gui/item_list.cpp

```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "item_list.h"
+#include "os/input.h"
+#include "os/keyboard.h"
```

Comment:
Those can be removed as you removed the backspace_pressed signal.

Could you also amend the commit message (`git commit --amend`) to remove mentions of what is actually not in this commit anymore?